README.md

```diff
@@ -1,2 +1,25 @@
 # orcharhino-scripts
 Utility scripts for the orcharhino-server. Might also work on foreman-/katello-servers.
+
+## Use at your own risk
+We try to consider all possible side-effects, while writing these scripts.
+Errors are always an option, though.
+So please follow these rules:
+
+1) create snapshots and/or backups of your system before running scripts that actively change data.
+1) read the script's documentation (README.md and/or run the script with `--help`), first.
```

Comment:
```suggestion
2. Read the script's documentation (`README.md` and/or run the script with `--help`), first.
```

Reply:
To github displaying the README.md, this makes no difference.
I usually use `1` for all lines, because it makes reordering much easier (should it be necessary).

Reply:
okay, valid point.